### ISSUE TITLE
PlayerHeight and NoteJumpOffset grayed out v1.7.1

### ISSUE THREAD

**Issue by KJRedTank** [RiskiVR/BSLegacyLauncher#56]
Just upgraded to v1.7.1 from 1.6.0 and suddenly Player Height and Note Jump Offset are grayed out in the menu. All the notes are spawning practically in my lap. 

I've tried adjusting Player Settings but the notes are still spawning right in front of me.

Did not have this issue in 1.6.0, and I have not changed any settings.

NOTE: Trying to get a screenshot of notes spawning quickly in front of you is not easy; put it on No Fail just to try to grab one that demonstrates the issue.
![20230713175957_1](https://github.com/RiskiVR/BSLegacyLauncher/assets/139514549/99c0d6f8-c2be-4670-a22c-f21058d18650)
![20230713180037_1](https://github.com/RiskiVR/BSLegacyLauncher/assets/139514549/13eb8612-adfa-4da0-8a97-03804dbf918f)

**Comment by RiskiVR** [comment]
Game bugs have nothing to do with launcher updates. Please check if you replaced the launcher files correctly.

**Comment by KJRedTank** [comment]
I've been replacing the launcher files for many versions now (since Beat Saber 1.15, whatever that was for BSLegacy Launcher), but I'll check again.

**Comment by KJRedTank** [comment]
Where can I download v1.6.0 **ZIP** file? 

I looked on the Discord and found the source code files but not the ZIP file. I followed the link from Discord to the release page for v1.6.0, but it's just the Source code files, which doesn't work.

Sorry I'm not a developer, so I don't really understand developer-speak. :/
![no zip](https://github.com/RiskiVR/BSLegacyLauncher/assets/139514549/cb7bb2a4-4582-46a8-9a62-08713ad762c8)

![current version download files](https://github.com/RiskiVR/BSLegacyLauncher/assets/139514549/cfc8535f-5fbf-4a28-82c4-d9b24fa0f165)

**Comment by RiskiVR** [comment]
Previous versions are not available for download.

**Comment by KJRedTank** [comment]
Then what do I do? I know you said it has nothing to do with the Launcher, but (1) it worked just fine with v1.6.0, (2) it stopped working as soon as I updated the Launcher to v1.7.1, (3) I tested v1.7.1 with both my installed versions of Beat Saber: 1.25.1 and 1.24. Same problem.

Here is a screenshot from Beat Saber 1.24:
![BeatSaber v 1 24](https://github.com/RiskiVR/BSLegacyLauncher/assets/139514549/11085311-27b0-4add-a352-fe3a1ddc4fb2)


Just so I can be clear, is there somewhere I can find STEP BY STEP instructions on how to replace (update) the BSLegacyLauncher files, OTHER THAN the video for initial install? (As I said, I've been using BSLegacyLauncher for many versions now, so I don't need to go through all the steps of connecting to Steam, downloading the previous Beat Saber version, etc. I've already done all that.)
